file-format-types.rdf

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+  xmlns:owl="http://www.w3.org/2002/07/owl#"
+  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+  xmlns:udfrs="http://www.udfr.org/onto#">
+  <owl:Ontology rdf:about="http://pcdm.org/file-format-types#">
+    <rdfs:comment xml:lang="en">PCDM File Format Genre ontology.</rdfs:comment>
+    <rdfs:label xml:lang="en">PCDM File Format Genres</rdfs:label>
+  </owl:Ontology>
+  <udfrs:GenreFacetType rdf:ID="Document">
```

Comment:
The udfr:Document description is very thin, and the nfo:Document description is only "A generic document. A common superclass for all documents on the desktop." -- so that's not much to go on.  So we could go with something a little more descriptive, such as "Documents produced by common desktop productivity software: text, markup, word processing, spreadsheet, presentation, etc."